bin/generate.js

```diff
@@ -0,0 +1,59 @@
+const { optimize } = require('svgo');
+const fs = require('fs');
+
+fs.mkdirSync('./dist/icons', { recursive: true });
+
+
+const ICONS_BUNDLE_FILE = './dist/main.js';
+
+// read all files from icons directory
+const files = fs.readdirSync('./src/icons');
+
+fs.writeFileSync(ICONS_BUNDLE_FILE, '');
+
+const TABLE_OF_ICONS = [
+    '| Icon | Name |',
+    '| --- | --- |'
+];
+
+for (let filename of files) {
+    const name = filename.split('.')[0];
+    const exportName = `Icon${name}`;
+
+    let iconCode = fs.readFileSync(`./src/icons/${filename}`, 'utf8');
+
+    const result = optimize(iconCode, {
+        plugins: [
+            'sortAttrs',
+            {
+                name: 'addAttributesToSVGElement',
+                params: {
+                    attributes: [
+                        {'fill': 'currentColor'}
+                    ]
+                }
+            }
```

Comment:
also, we need to pass `prefer viewBox` (can't remember the exact name of the option)